GitOps.md

```diff
@@ -25,7 +25,8 @@ helm upgrade --install orchestrator-gitops . -f values.yaml -n orchestrator-gito
 To allow the Tekton resources to push to the registry, we need an account capable to push the image to the registry:
 
 * Create or edit a [Robot account](https://quay.io/organization/orchestrator?tab=robots) and grant it `Write` permissions to the newly created repository
-* Download `the Docker configuration` file for the robot account and move it under the root folder of this repository (we assume the file name is `orchestrator-orchestror-ci-auth.json`)
+* Download the `Docker configuration` file for the robot account and move it under the root folder of this repository (we assume the file name is `orchestrator-orchestror-ci-auth.json`)
+  * TODO: you may grant permission
```

Comment:
permission to Read the file? I don't think it's needed

Reply:
yes it is! I had to ask for the correct permission otherwise I could see an empty box :)

Reply:
so, what `permission` you need to define? I thought it was a file permission, but I guess you meant something else

Reply:
Please check the updated version of this sentence.

I'm referring to quay.io permissions to see this 
![image](https://github.com/parodos-dev/orchestrator-helm-chart/assets/15085664/178c0fb8-7bd6-4608-9cb9-0ce6a3e5f2c4)
and this 
![image](https://github.com/parodos-dev/orchestrator-helm-chart/assets/15085664/20558cdf-ef10-4dde-adad-172beffc34af)
and thus to download the dockerfile.